src/cms/views/events/event_list_view.py

```diff
@@ -33,6 +33,27 @@ class EventListView(LoginRequiredMixin, PermissionRequiredMixin, TemplateView):
     template_archived = "events/event_list_archived.html"
     #: Whether or not to show archived events
     archived = False
+    #: Messages in confirmation dialogs for delete, archive, restore operations
+    confirmation_dialog_context = {
+        "archive_dialog_title": ugettext_lazy(
+            "Please confirm that you really want to archive this event"
+        ),
+        "archive_dialog_text": ugettext_lazy(
+            "All translations of this event will also be archived."
+        ),
+        "restore_dialog_title": ugettext_lazy(
+            "Please confirm that you really want to restore this event"
+        ),
+        "restore_dialog_text": ugettext_lazy(
+            "All translations of this event will also be restored."
+        ),
+        "delete_dialog_title": ugettext_lazy(
+            "Please confirm that you really want to delete this event"
+        ),
+        "delete_dialog_text": ugettext_lazy(
+            "All translations of this event will also be deleted."
+        ),
+    }
```

Comment:
If you have context you want to inject into multiple views, you should do this with the [ContextMixin](https://docs.djangoproject.com/en/2.2/ref/class-based-views/mixins-simple/#contextmixin):

`event_mixin.py`
```
from django.utils.translation import ugettext as _
from django.views.generic.base import ContextMixin

class EventMixin(ContextMixin):
    extra_context = {
        ...
    }
```

`event_list_view.py` and `event_view.py`
```
from .event_mixin import EventMixin

class EventListView(LoginRequiredMixin, PermissionRequiredMixin, TemplateView, EventMixin):
# or
class EventView(PermissionRequiredMixin, TemplateView, EventMixin):

...

        context = self.get_context_data(**kwargs)
        return render(
            request,
            self.template_name,
            {
                **context,
                ...
            } 
```
This also applies to the other content types.

Reply:
When I was preparing the content types for mixins and while I was testing the context rendering, I experienced a problem on develop branch with the page form. The form does no long detects, when the form is saved with no changes. This was earlier displayed by a message. Do you know, whats going on there?

Reply:
Yes, I already opened the issue #466 (Page form changes not correctly detected) some time ago. You can ignore this in your PR.

Reply:
Thanks for the info, i'm just surprised because when I was fixing the icon upload bug, this 'no changes' message occurred to me. Seems like it works occasionally.